arboretum/technology/auditree/README.md

```diff
@@ -1,26 +1,31 @@
 # Auditree technology library
 
-The fetchers and checks contained within this `auditree` technology folder are common tests that can be configured and
-executed for the purpose of generating compliance reports and notifications using the [auditree-framework][].
-See [auditree-framework documentation](https://complianceascode.github.io/auditree-framework/) for more details.
+The fetchers and checks contained within this `auditree` technology folder are
+common tests that can be configured and executed for the purpose of generating
+compliance reports and notifications using the [auditree-framework][].
```

Comment:
Maybe say something about how these are used to validate the configuration and ensure smooth running of an auditree instance ?

Reply:
Done in https://github.com/ComplianceAsCode/auditree-arboretum/pull/5/commits/414f4a9ecf743dc20c2ec716457b24bf01be07f5